tests/lib/rules/jsx-sort-props.js

```diff
@@ -202,9 +202,19 @@ ruleTester.run('jsx-sort-props', rule, {
       errors: [expectedError]
     },
     {
+      code: '<App key={2} b a />',
+      options: reservedFirstAsBooleanArgs,
+      errors: [expectedError]
+    },
+    {
+      code: '<App b a />',
+      options: reservedFirstAsBooleanArgs,
+      errors: [expectedError]
+    },
+    {
       code: '<App dangerouslySetInnerHTML={{__html: "EPR"}} key={2} b />',
       options: reservedFirstAsBooleanArgs,
-      errors: [expectedReservedFirstError]
+      errors: [expectedReservedFirstError, expectedError]
```

Comment:
I think that after the changes this is triggering the `expectedError` as the `||
 (!previousIsReserved && !currentIsReserved)` check and it is true when evaluating `dangerouslySetInnerHTML` and `b`.